### ISSUE TITLE
Broken acrylic in htop on latest preview.

### ISSUE THREAD

**Issue by r3core** [microsoft/terminal#7099]
# Environment

```none
Windows build number: Microsoft Windows NT 10.0.19041.0
Windows Terminal version (if applicable): 1.2.2022.0 (Preview) and 1.1.2021.0 (Stable)
Ubuntu build number: Ubuntu 20.04 LTS on WSL2

Any other software?
htop 2.2.0(Black Night theme)
```

# Steps to reproduce

1. Open preview Terminal (1.2.2022.0)
2. Configure a WSL profile with acrylic effect enabled. (Mine is the following)
```jsonc
            {
                "guid": "{07b52e3e-de2c-5db4-bd2d-ba144ed6c273}",
                "hidden": false,
                "name": "Ubuntu-20.04",
                "source": "Windows.Terminal.Wsl",
                "commandline" : "wsl.exe ~",
                "fontFace": "Cascadia Code PL",
                "fontSize" : 8,
                "acrylicOpacity" : 0.5,
                "useAcrylic" : true,
                "colorScheme": "Tomorrow Night Burns", // Issue is present with other colour themes as well.
                "scrollbarState": "hidden"
            }
```
3. Open `htop`
4. Set `htop` to _Black Night_ theme (F2 Setup -> Colors -> Black Night).
5. Note acrylic is broken.
6. Re-run above steps on the stable terminal (1.1.2021.0)
8. Note acrylic is working as expected.
# Expected behavior

In the stable Terminal, acrylic working as expected whether opened directly or within a tmux session, hence why I bothered with creating an issue.
![stablefullhtop](https://user-images.githubusercontent.com/37566423/88643187-bf6eb780-d104-11ea-8d51-586115fe8d50.png)

![stabletmuxhtop](https://user-images.githubusercontent.com/37566423/88643201-c39ad500-d104-11ea-8d8c-87d60d2af5b5.png)
# Actual behavior

The preview Terminal's htop's acrylic is broken either way. Same tmux session was used for both screenshots.
![previewfullhtop](https://user-images.githubusercontent.com/37566423/88643263-d3b2b480-d104-11ea-95a6-16184d73de53.png)

![previewtmuxhtop](https://user-images.githubusercontent.com/37566423/88643278-d7463b80-d104-11ea-92d4-aeab6a1ab573.png)

**Comment by j4james** [comment]
When the acrylic option is enabled, only the default background color is transparent. But the _Black Night_ theme in `htop` sets the background color to black (which isn't meant to be transparent). So the latest version of Terminal is actually working as intended.

The reason why it was transparent in earlier versions of Terminal was because of a bug in conpty that converted black background colors to the default color, and that has now been fixed.

The bottom line is if you want the screen to be transparent, you need to use a theme with a _default_ background color.

**Comment by r3core** [comment]
Right. Thanks for the clarification.
Will close this. 👍
